lib/store/index.js

```diff
@@ -39,23 +39,30 @@ export class Store {
 
   @computed
   get kernel(): ?Kernel {
+    if (!this.grammar || !this.editor) return null;
+
     if (this.globalMode) {
-      if (!this.grammar) return null;
       const currentScopeName = this.grammar.scopeName;
-      return this.runningKernels.filter(
+      return this.runningKernels.find(
         k => k.grammar.scopeName === currentScopeName
-      )[0];
+      );
     }
-
-    if (!this.filePath) return null;
+    //
```

Comment:
Empty comment?

Reply:
Invisible comment 🧙‍♂️. We can remove it with magic.